### PR TITLE
Lvnodcdxi 32

### PR DESCRIPTION
**Pull request by StanislavFedyna**
realizer composition for additional config component (prepared for styling hole project)

### CODE REVIEW

File: src/client/components/additionalConfigs/additionalConfigs.js [hunk 1]
```diff
@@ -1,30 +1,16 @@
...
+import Settings from './settings/settings';
+import VolumeSlider from './volumeSlider/volumeSlider';
 
 import './additionalConfigs.css';
 
-const volumeIcon = <FontAwesomeIcon icon={faVolumeUp} />;
-const cogIcon = <FontAwesomeIcon icon={faCog} />;
-
 const AdditionalConfigs = () => {
   return (
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/additionalConfigs.js]: You can avoid return statement here

File: src/client/components/additionalConfigs/modesThemeSwitcher/modesThemeSwitcher.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './modesThemeSwitcher.css';
+
+const ModesThemeSwitcher = () => {
+  return (
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/modesThemeSwitcher/modesThemeSwitcher.js]: You can avoid return statement in all components where you have nothing before this statement

File: src/client/components/additionalConfigs/volumeSlider/volumeSlider.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import Slider from 'react-rangeslider';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faVolumeUp } from '@fortawesome/free-solid-svg-icons';
+import 'react-rangeslider/lib/index.css';
+import './volumeSlider.css';
+
+const volumeIcon = <FontAwesomeIcon icon={faVolumeUp} />;
+
+class VolumeSlider extends React.Component {
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/volumeSlider/volumeSlider.js]: React.Component can be destructed from react

File: src/client/components/additionalConfigs/settings/settings.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import './settings.css';
+
+const settingsIcon = <FontAwesomeIcon icon={faCog} />;
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/settings/settings.js]: I would suggest to create a file with all icons for your project and import it from there

File: src/client/components/additionalConfigs/volumeSlider/volumeSlider.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    super(props, context);
+    this.state = {
+      value: 20,
+      maxValue: 100,
+      minValue: 0
+    };
+  }
+  handleOnChange = (value) => {
+    this.setState({
+      value: value
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/volumeSlider/volumeSlider.js]: In es6 you can avoid writing key: value in object literals and write just key if they have same names

File: src/client/components/additionalConfigs/volumeSlider/volumeSlider.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+class VolumeSlider extends React.Component {
+  constructor (props, context) {
+    super(props, context);
+    this.state = {
+      value: 20,
+      maxValue: 100,
+      minValue: 0
+    };
+  }
+  handleOnChange = (value) => {
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/volumeSlider/volumeSlider.js]: Better to name your function parameters, constants, etc more informatively, for eg., sliderValue

File: src/client/components/additionalConfigs/settings/settings.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {settingsIcon} from '../../../assets/icons/icons';
+import './settings.css';
+
+const Settings = () => {
+  return (
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/settings/settings.js]: Here return statement can be removed too

Comment (StanislavFedyna) [reply src/client/components/additionalConfigs/settings/settings.js]: sorry, my bad, missed first time
done)

**Review by tonypntsrn — APPROVED** [review]

**Review by MaxVoloskiy — APPROVED** [review]

**Pull request merged**